gossip/gossip.go

```diff
@@ -428,23 +424,15 @@ func (g *Gossip) manage() {
 		if g.filterExtant(g.bootstraps).len() > 0 || (g.bootstraps.len() == 0 && len(g.gossipBootstrap) > 0) {
 			if g.outgoing.len()+g.incoming.len() == 0 {
 				log.Infof("no connections; signaling bootstrap")
-				g.stalled.Signal()
+				g.stalled <- struct{}{}
```

Comment:
Is there any danger of the stalled channel reaching its 10-entry limit? If so, we should put this in a select to make the write non-blocking. We could also increase the limit here (why 10 instead of 1?); a high capacity on a `chan struct{}` is free.